flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/calls/ScalarOperatorGens.scala

```diff
@@ -936,6 +936,12 @@ object ScalarOperatorGens {
       operand: GeneratedExpression,
       targetType: LogicalType)
     : GeneratedExpression = {
+
+    ctx.addReusableHeaderComment("Configuration")
+    ctx.addReusableHeaderComment("-------------")
```

Comment:
Perhaps move these two directly in `getClassHeaderComment`?

Reply:
It feels more generic to "limit" other usages to this "Configuration" header.
I'd rather do this if/when we get a second usage of the header, but if you prefer to have it there already also fine, I can move.